shared/bls/spectest/fast_aggregate_verify_test.yaml.go

```diff
@@ -7,6 +7,7 @@ type FastAggregateVerifyTest struct {
 	Input struct {
 		Pubkeys   []string `json:"pubkeys"`
 		Message   string   `json:"message"`
+		Messages  string   `json:"messages"`
```

Comment:
Same comment here about a TODO or comment. This should be removed after https://github.com/ethereum/eth2.0-spec-tests/issues/22 is resolved.